File: cs230/cs230-dz07-nikola_tasic_3698/reverse/src/main/java/rs/ac/metropolitan/cs230/entity/Role.java
package rs.ac.metropolitan.cs230.entity;

import javax.persistence.*;

import java.time.LocalDate;
import java.util.*;

@Entity
@Table(name = "role")
public class Role {
	@Id
	@GeneratedValue(strategy = GenerationType.IDENTITY)
	@Column(name = "id_role")
	public Long idRole;

	@Column(name = "role_name")
	public String roleName;

	public Role() {}
	public Long getIdRole() {
		return idRole;
	}

	public void setIdRole(Long idRole) {
		this.idRole = idRole;
	}


	public String getRoleName() {
		return roleName;
	}

	public void setRoleName(String roleName) {
		this.roleName = roleName;
	}

}

File: cs322/cs322-pz-nikola_tasic_3698/blog-backend/Data/Entity/Role.cs
namespace blog_backend.Data;

public class Role {
	private string name;

	public string Name {
		get => name.Replace("ROLE_", "");
		set => name = "ROLE_" + value.Replace("ROLE_", "");
	}

	public Role(string name) {
		this.name = name;
	}
}